Timeseries to table transformation: Improve Documentation (#83647)
grafana/grafana: docs/sources/panels-visualizations/visualizations/table/index.md

# Table

> Tables are very flexible, supporting multiple modes for time series and for tables, annotation, and raw JSON data. This visualization also provides date formatting, value formatting, and coloring options. In addition to formatting and coloring options, Grafana also provides a variety of _Cell types_ which can be used to display gauges, sparklines, and other rich data displays.

Reviewer: We want to avoid passive voice as it's less clear than active voice. I've made the same type of change to the other additions.

```suggestion
Tables are very flexible, supporting multiple modes for time series and for tables, annotation, and raw JSON data. This visualization also provides date formatting, value formatting, and coloring options. In addition to formatting and coloring options, Grafana also provides a variety of _Cell types_ which you can use to display gauges, sparklines, and other rich data displays.
```

### Sparkline

> Shows value rendered as a sparkline. Sparklines can be shown by taking data with multiple time series and using the [Time series to table transformation][] to process data to a format the table can show.

Reviewer: ```suggestion
Shows value rendered as a sparkline. You can show sparklines by using the [Time series to table transformation][] on data with multiple time series to process it into a format the table can display.
```

[calculations]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations/query-transform-data/calculation-types"
[calculations]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/visualizations/panels-visualizations/query-transform-data/calculation-types"

[Time series to table transformation]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations/query-transform-data/transform-data#time-series-to-table-transform"
[Time series to table transformation]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations/query-transform-data/transform-data#time-series-to-table-transform"

> [Time series panel]: "docs/grafana/" --> /docs/grafana/<GRAFANA VERSION>/panels-visualizations/visualizations/time-series/
> [Time series panel]: "docs/grafana-cloud/" --> /docs/grafana/<GRAFANA VERSION>/panels-visualizations/visualizations/time-series/
> 
> [color scheme]: "docs/grafana/" --> docs/grafana/<GRAFANA VERSION>/panels-visualizations/configure-standard-options#color-scheme
> [color scheme]: "docs/grafana-cloud" -> docs/grafana/<GRAFANA VERSION>/panels-visualizations/configure-standard-options#color-scheme

Reviewer: There are some formatting errors here. I've also swapped the Cloud links to link to Cloud docs and future-proofed the version interpolation syntax.
```suggestion
[Time series panel]: "/docs/grafana/ -> /docs/grafana/<GRAFANA_VERSION>/panels-visualizations/visualizations/time-series"
[Time series panel]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/visualizations/panels-visualizations/visualizations/time-series"

[color scheme]: "/docs/grafana/ -> /docs/grafana/<GRAFANA_VERSION>/panels-visualizations/configure-standard-options#color-scheme"
[color scheme]: "/docs/grafana-cloud -> /docs/grafana-cloud/visualizations/panels-visualizations/configure-standard-options/#color-scheme"
```

Author: Thanks for the changes Isabel!
